executor/mem_reader.go

```diff
@@ -14,6 +14,8 @@
 package executor
 
 import (
+	"time"
```

Comment:
Is this change required for this PR? It doesn't seem like it is related.

Reply:
I found a panic/crashing bug when I added the circular/atomic RENAME USER swap test, which I fixed in this PR as well. Should I have that fix in a separate PR? And disable (comment out) the failing test?

Reply:
I think this should use `m.ctx.GetSessionVars.Location()` instead of `m.ctx.GetSessionVars.TimeZone`.

See the function here:

https://github.com/pingcap/tidb/blob/f13f696400be61ae43dd28aeaf61920817732b17/sessionctx/variable/session.go#L1308-L1315

Reply:
Looks like it can no longer be reproduced. I am going to close this conversation.